lib/content-services/src/lib/tag/services/tag.service.ts

```diff
@@ -33,16 +33,19 @@ import {
 @Injectable({
     providedIn: 'root'
 })
-// eslint-disable-next-line @angular-eslint/directive-class-suffix
 export class TagService {
 
-    _tagsApi: TagsApi;
+    private _tagsApi: TagsApi;
     get tagsApi(): TagsApi {
         this._tagsApi = this._tagsApi ?? new TagsApi(this.apiService.getInstance());
         return this._tagsApi;
     }
 
-    private searchApi: SearchApi = new SearchApi(this.apiService.getInstance());
+    private _searchApi: SearchApi;
```

Comment:
I know it is merged PR but for future - I'm wondering if that is proper approach. SearchAPI is used only by TagService internally. Additionally why somebody would like to refer to searchApi from TagService object? Getter for that seems useless because it will be used only by unit tests (we probably should not modify code only because of unit tests). It can be still mocked out in unit tests for example by spying prototype.